pom.xml

```diff
@@ -1773,6 +1773,35 @@
       <id>thriftif</id>
       <build>
         <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>1.2.1</version>
```

Comment:
Seems the version is different that the one being used in hive
https://github.com/apache/hive/blob/81909f3ed294a5c095797346d26d1d1917d42c3a/standalone-metastore/pom.xml#L43

Good if we can extract the variable and use the same variable at all places

Reply:
Thanks Ayush for noticing this, unfortunately the `standalone-metastore` pom does not inherit from the topmost one, so we can't share it, but I will introduce a variable with the same name and use the same version.

I have tried locally, everything seems good (normal build and thrift generation, OK and KO paths).